docs/rules/jsx-first-prop-new-line.md

```diff
@@ -69,6 +75,27 @@ The following patterns are not considered warnings when configured `"multiline"`
 />
 ```
 
+The following patterns are considered warnings when configured `"multiline-multiprop"`:
+
+```jsx
+<Hello foo={{
+    }}
+    bar />
+```
+
+The following patterns are not considered warnings when configured `"multiline-multiprop"`:
+
+```jsx
+<Hello foo={{
```

Comment:
wait, why is this not a warning? The JSX tag is spanning multiple lines, so it should look like:
```jsx
<Hello
  foo={{
    …
  }}
/>
```